CHANGELOG.md

```diff
@@ -46,6 +47,8 @@ Changelog for FrostFS Node
 - Actually use `object.put.pool_size_local` and independent pool for local puts (#64).
 
 ### Removed
+- Parameter `name` from command `frostfs-cli container create` (#37)
```

Comment:
We don't need to remove it, it is still valid, just mean a different thing (application-level readable identifier).

Reply:
Oops, reverted.